src/templates/Navigator/index.js

```diff
@@ -1,16 +1,45 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
 import styles from './navigator.module.scss';
 import classnames from 'classnames';
 
 function Navigator ( props ) {
   const {
     children,
-    className
+    className,
+    txt
   } = props;
 
   return (
     <div className={ classnames(className, styles['navigator']) }>
-      { children }
+      <div className={ styles['left-rail'] }>
+        <Icon
+          icon='pgLogo'
+          className={ styles.logo }
+        />
+        {
+          txt &&
+          <TextComponent
+            type={ 3 }
+            className={ styles['page-txt'] }
+            light
+          >
+            { txt }
+          </TextComponent>
+        }
+      </div>
+      <div className={ styles.main }>
+        <StepProgress
+          className={ styles['step-progress'] }
+          data={[
+            { active: false, inactive: false, icon: 'icon-calculator', label: 'Aenean', link: '#' },
```

Comment:
a bit unrelated, but I think `StepProgress` should have `onClick` handlers for each item. Would make this component more flexible for the end user